### PR TITLE
Reactor

### PR DESCRIPTION
**Pull request by mattheworiordan**
@SimonWoolf @paddybyers @ashleyf1 I had to create new pages for the three Reactor offerings, and whilst I was at it, made some updates to the smart notifications page.

I am now going to merge this into the main website, but it would be useful if you have a chance to read through as I expect it has typos & inconsistencies (it's 2am and I am in a bit of a rush).

Would you mind having a quick look over for obvious problems?

### CODE REVIEW

**Comment by SimonWoolf** [comment]
Looks good. Added 196b775 with a few minor tweaks.

File: content/general/queues.textile [hunk 1]
```diff
@@ -28,7 +28,7 @@ Our multi-tenanted queue service is provided free as part of the core platform t
 
 For customers with more demanding requirements (up to millions of messages per second), Ably provides dedicated queue clusters as part of our "Enterprise offering":https://www.ably.io/pricing/enterprise. "Get in touch":https://www.ably.io/contact if you would like to discuss our dedicated queue service.
 
-Alternatively, for Enterprise customers, we also offer our "Ably Reactor Firehose":https://www.ably.io/reactor. Instead of consuming data from our queues, we will stream your realtime data directly into a third party streaming or queuing service. "Find our more about the Reactor Firehose":https://www.ably.io/reactor.
+Alternatively, for Enterprise customers, we also offer our "Ably Reactor Firehose":/general/firehose, where instead of consuming data from our queues, we will stream your realtime data directly into a third party streaming or queuing service.
```
Comment (paddybyers) [inline content/general/queues.textile]: typo queuing -> queueing

File: content/general/queues.textile [hunk 2]
```diff
@@ -4,13 +4,13 @@ section: general
...
+Using configurable rules, you can ensure that your messages, presence and channel lifecycle events are enqueued on an Ably message queue. Unlike our "channels":/realtime/channels-messages#channels which follow a "pub/sub pattern":https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern where each message is delivered to any number of subscribers, our "message queues":https://en.wikipedia.org/wiki/Message_queue operate on the basis that each messages is delivered to only one subscriber. Consuming from Ably queues is done using the popular "AMQP":https://www.amqp.org/ or "STOMP":https://stomp.github.io/ protocols.
```
Comment (paddybyers) [inline content/general/queues.textile]: typo "each messages is delivered" -> "each message is delivered"

File: content/general/firehose.textile [hunk 3]
```diff
@@ -4,14 +4,14 @@ section: general
 index: 26
 ---
 
-Our Firehose can stream your realtime data published within the Ably platform directly to another streaming or queueing service. For example, all messages published by any device on a channel could be immediately streamed to Amazon Kinesis allowing you to process this data in realtime. Firehose is offered as part of our "Ably Reactor service":https://www.ably.io/reactor, however it is only available to Enterprise customers.
+Our Firehose can stream your realtime data published within the Ably platform directly to another streaming or queueing service. For example, all messages published by any device on a channel could be immediately streamed to Amazon Kinesis allowing you to process this data in realtime. Firehose is offered as part of our "Ably Reactor service":https://www.ably.io/reactor, however it is only available to "Enterprise customers":https://www.ably.io/pricing/enterprise.
```
Comment (paddybyers) [inline content/general/firehose.textile]: > Firehose is offered as part of our "Ably Reactor service":https://www.ably.io/reactor, however it is only available to "Enterprise customers":https://www.ably.io/pricing/enterprise.

Grammatical nitpick: pls don't join two sentences with a comma; use  semicolon or say, for example,

Firehose is offered as part of our "Ably Reactor service":https://www.ably.io/reactor but is currently only available to "Enterprise customers":https://www.ably.io/pricing/enterprise.

**Comment by mattheworiordan** [comment]
Thanks for feedback & lots of fixes @SimonWoolf!

**Pull request merged**